### DISCUSSION TITLE
part-ii/2010-151/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#349] [Question comments]
Part II 2010 Topics in Analysis: Paper 3, Section II, F 

https://questions.tripos.org/part-ii/2010-151/

**Comment by DannyDuberstein** [comment]
(iii), (iv):  "dog walking" extends to "<=" provided you avoid 0, as a branch of Log is well-defined on $\mathbb{C} - \mathbb{R_{\leq 0}}$
